pyne/partisn.py

```diff
@@ -472,36 +454,19 @@ def _check_fine_mesh_total(block01):
         if key in ['it', 'jt', 'kt']:
             total += block01[key]
     
-    if total >= 7:
-        # no warning necessary
-        return False
-    else:
-        # warn the user
-        return True
+    if total < 7:
```

Comment:
I recall getting error messages to that nature but I did not thoroughly investigate. In practice I don't think it will come up much; any real problem will be plenty more then 7 fine mesh intervals.